From foo@baz Thu Jan 17 09:33:34 CET 2019
Date: Thu, 17 Jan 2019 09:33:34 +0100
To: Greg KH <gregkh@linuxfoundation.org>
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: spdxcheck: print out files without any SPDX lines.

Just a hack to make it easy to see what still needs to be converted.



---
 scripts/spdxcheck.py |    3 +++
 1 file changed, 3 insertions(+)

--- a/scripts/spdxcheck.py
+++ b/scripts/spdxcheck.py
@@ -204,7 +204,10 @@ def scan_git_tree(tree):
         if not os.path.isfile(el.path):
             continue
         with open(el.path, 'rb') as fd:
+            i = parser.spdx_valid
             parser.parse_lines(fd, args.maxlines, el.path)
+            if i == parser.spdx_valid:
+                sys.stdout.write('%s\n' %el.path)
 
 def scan_git_subtree(tree, path):
     for p in path.strip('/').split('/'):
